_raml/apps/legal/schema/get_legal.schema.json

```diff
@@ -4,21 +4,27 @@
   "type": "object",
   "properties": {
     "contactInformation": {
+      "type": "string",
```

Comment:
for all properties except link:
"type": "[contentPageSummary](page:apps-data-types#contentPageSummary)",

where *contentPageSummary* is a new datatype with only one property: 

    "default": {
      "name": "string",
      "description": "The name of the content page."
    },